src/contrib/cluster/Akka.Cluster.Sharding/Akka.Cluster.Sharding.csproj

```diff
@@ -5,9 +5,9 @@
     <AssemblyTitle>Akka.Cluster.Sharding</AssemblyTitle>
     <Description>Sharded actors with managed lifecycle for Akka.NET cluster</Description>
     <VersionSuffix>beta</VersionSuffix>
-    <TargetFrameworks>$(NetFrameworkLibVersion);$(NetStandardLibVersion)</TargetFrameworks>
     <PackageTags>$(AkkaPackageTags);network;cluster;sharding</PackageTags>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
Need to use the parameterized values from common.props, and the attribute will still need to be `<TargetFrameworks>` since we'll likely be targeting multiple versions of .NET Standard or .NET Standard + .NET Core + .NET Framework, depending on https://github.com/akkadotnet/akka.net/issues/3851

Reply:
I did it mostly because Microsoft.Extensions.DependencyInjection is not compatible with .NET Framework pre 4.6.1. I can set it back up.

Reply:
My recommendation: if we're going to drop .NET framework 4.5.2 (good chance) definitely do it in a separate PR - don't couple it in this one